deeplinkdispatch-processor/src/main/java/com/airbnb/deeplinkdispatch/DeepLinkProcessor.java

```diff
@@ -163,9 +190,7 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
       if (deepLinkAnnotation != null) {
         deepLinks.addAll(Arrays.asList(deepLinkAnnotation.value()));
       }
-      if (customAnnotatedElements.contains(element)) {
-        deepLinks.addAll(enumerateCustomDeepLinks(element, prefixes));
-      }
+      deepLinks.addAll(enumerateCustomDeepLinks(element, prefixes));
```

Comment:
By removing this check, it's more likely that the user will receive the message stating that their annotation has not been registered.